common/source/docs/common-temperature-sensor.rst

```diff
@@ -5,54 +5,60 @@
-
-ArduPilot already has numerous possible sources for temperature reports: ESCs, Smart Batteries, Motor EFI and these independent sensors can be used to replace those devices temperature reports which already exist in ArduPilot. They can also just be logged.
+ArduPilot already has numerous possible sources for temperature reports: ESCs, Smart Batteries, Motor EFI and these independent sensors can be used to replace those device's temperature reports that already exist in ArduPilot. They can also just be logged.
 
 I2C Sensor Setup
 ================
 
-Attach sensor to one of the autopilot's I2C ports using SDA and SCL pins. Then set (examples shown for first sensor):
+Attach the sensor to one of the autopilot's I2C ports using SDA and SCL pins. Then set (examples shown for the first sensor):
+
+- :ref:`TEMP1_TYPE<TEMP1_TYPE>` = 0:Disabled, 1:TSYS01, 2:MCP9600, 3:MAX31865, 4: TSYS03
+- :ref:`TEMP1_BUS<TEMP1_BUS>` = the I2C port on which the sensor is attached (0-3).
+- :ref:`TEMP1_ADDR<TEMP1_ADDR>` = the sensor's I2C address (0-127).
+- :ref:`TEMP1_SRC<TEMP1_SRC>` = the temperature reporting device type that should have its temperature report replaced with this sensor's.
+- :ref:`TEMP1_SRC_ID<TEMP1_SRC_ID>` = identifies the individual device, of the above type, that should have its report replaced.
 
-- :ref:`TEMP1_TYPE<TEMP1_TYPE>` = 0:Disabled, 1:TSYS01, 2:MCP9600, 3:MAX31865, 4: TSYS03 Sensor type
-- :ref:`TEMP1_BUS<TEMP1_BUS>` = 0-3 Board's I2C port on which the sensor is attached
-- :ref:`TEMP1_ADDR<TEMP1_ADDR>` = the sensor's I2C address (0-127)
-- :ref:`TEMP1_SRC<TEMP1_SRC>` = which other temperature reporting device type should have its temperature report replaced with this sensor's.
-- :ref:`TEMP1_SRC_ID<TEMP1_SRC_ID>` = this identifies which individual device, of the above type, should have its report replaced.
+.. note:: Each I2C device on any single bus must have a unique address. If more than one of the same type of I2C temperature sensor is used, ensure each is configured with a unique address. Some breakout boards include jumpers for this purpose. Refer to the product datasheet for specific instructions regarding I2C address configuration.
+
+.. note:: The MAX31865 is an SPI device, which uses the same parameters and basic configuration procedure as I2C sensors but must be connected to an SPI port rather than an I2C port. Some autopilots do not expose an external SPI port, in which case the MAX31865 cannot be easily used.
 
 Analog Sensor Setup
 ===================
 
-Atttch the sensor to an analog input pin. The pin number can be determined from the autopilot's wiki page. Then set (examples shown for first sensor):
+Attach the sensor to an analog input pin (an unused battery voltage or current monitor pin, for example). The pin number can be determined from the autopilot's wiki page. Use care not to exceed the pin's rated voltage (typically 3v3) with the sensor output. Then set (examples shown for the first sensor):
```

Comment:
```suggestion
Attach the sensor to an analog input pin (an unused rssi input, analog airspeed input, battery voltage or current monitor input pin, for example). The pin number can be determined from the autopilot's wiki page. Use care not to exceed the pin's rated voltage (typically 3v3) with the sensor output. Then set (examples shown for the first sensor):
```

Reply:
Also, probably best to use "3.3 V" instead of "3v3" which might be confusing to people not used to this short-hand form.

Reply:
Fixed both.